lib/color-schemes/Falcon.colorscheme

```diff
@@ -1,121 +1,126 @@
-# Author: Alberto Salvia Novella (es20490446e)
 
+### GENERAL
+[General]
+Description=Falcon
+Author=Alberto Salvia Novella (es20490446e)
+Opacity=1
+
+### BACKGROUND (#223333)
 [Background]
-Bold=false
 Color=34,51,51
+Bold=false
 Transparency=false
 
+### BACKGROUND INTENSE (#223333)
 [BackgroundIntense]
-Bold=true
 Color=34,51,51
+Bold=true
 Transparency=false
 
+### FOREGROUND (#C2C2C2)
 [Foreground]
-Bold=false
 Color=194,194,194
+Bold=false
 Transparency=false
 
+### FOREGROUND INTENSE (#C2C2C2)
 [ForegroundIntense]
-Bold=true
 Color=194,194,194
+Bold=true
 Transparency=false
 
-## BLACK
+### BLACK (#959595)
```

Comment:
Let's keep two # symbols for consistency.

Reply:
Fixed 👍

I chose to keep three for all, because with two there wasn't enough distinction between the title and the rest of data in a section.

Reply:
> Fixed 👍
> 
> I chose to keep three for all, because with two there wasn't enough distinction between the title and the rest of data in a section.

No problem, that's also good!